mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -1113,13 +1152,34 @@ func expandLayer7DdosDefenseConfig(configured []interface{}) *compute.SecurityPo
 	}
 }
 
+<% unless version == 'ga' -%>
+func expandAutoDeployConfig(configured []interface{}) *compute.SecurityPolicyAdaptiveProtectionConfigAutoDeployConfig {
+	if len(configured) == 0 || configured[0] == nil {
+		return nil
+	}
+
+	data := configured[0].(map[string]interface{})
+
+	return &compute.SecurityPolicyAdaptiveProtectionConfigAutoDeployConfig{
+		LoadThreshold:   data["load_threshold"].(float64),
+		ConfidenceThreshold:   data["confidence_threshold"].(float64),
+		ImpactedBaselineThreshold:   data["impacted_baseline_threshold"].(float64),
+		ExpirationSec:   int64(data["expiration_sec"].(int)),
+		ForceSendFields: []string{"Enable"},
```

Comment:
Since `Enable` is not a field in `SecurityPolicyAdaptiveProtectionConfigAutoDeployConfig`, this setting won't do anything. You may not need any `ForceSendFields`, it would depend if the API defaults to 0 for these number fields, or if you want 0's to be explicitly sent.

Reply:
Hello there! We removed this part from the code but we get some issues from VCR-test. I think we've to keep this config, the provider `layer_7_ddos_defense_config` has the same configuration and both are in the same context. We decided to go back with this part on the code.

Reply:
@afpesseti sorry for the confusion with the VCR tests. It looks like you got an unrelated failure from a test that has been flaky for us: `TestAccCloudfunctions2function_cloudfunctions2BasicGcsExample`. Could you try removing the `ForceSendFields` again? I suspect it will work as expected for this particular resource and its tests.

Reply:
Ok, a removed again :D I think it will work, I made local test before push the updates. Thank so much again!